src/components/Dropdown/dropdown.module.scss

```diff
@@ -1,44 +1,59 @@
-$dropdown-hover: #d7d6d6;
+$dropdown-hover: #d0e7ff;
 $white: #fff;
 $theme-blue: #1d1283;
 $theme-active: #87d870;
 $black: #000;
 
 .dropdown {
-    width: 180px;
-    background-color: $white;
-    border: 2px solid $black;
-    border-radius: 0.5rem;
-    color: $black;
+    width: 150px;
```

Comment:
Will it cause any issue, if we convert this into REM ?

Reply:
no it wont. But as you can see in the video, it is responsive still